build/helper/documentation_helper.py

```diff
@@ -631,7 +637,7 @@ def test_get_function_rst():
 
             
 ''' # noqa
-    assert actual_function_rst == expected_fuction_rst
+    assert _remove_trailing_whitespace(actual_function_rst) == _remove_trailing_whitespace(expected_fuction_rst)
```

Comment:
I think it would be more useful to compare line by line. That way the error can pinpoint exactly where the mismatch is. That also lets the comparison be more correct as far as what matters to rst. Like 1 empty line is the same as 42 empty lines.

Reply:
ok